microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -41,6 +41,8 @@ We are aware that Chromium has revised the timelines for Manifest V2 sunset. We
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |
 
+MV2 extensions will continue to be supported through enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024).
```

Comment:
MV2 extensions will continue to be supported through the enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024).